views/base.twig

```diff
@@ -1,7 +1,9 @@
+{% set title = 'App Name' %}
```

Comment:
I like this much more than serving it as a param to twig from the router! Is it possible to override this `title` in different twig files that extend base.twig? For example, in the dashboard twig file, it would be nice to have the title be `Dashboard - App name`.

Reply:
Yeah the variable is easily overloaded! That's one of the benifits

Reply:
Sweet :sunglasses: